### ISSUE TITLE
Consultation on code training results

### ISSUE THREAD

**Issue by Christmas-TR** [Mehooz/vision4leg#16]
Hello, I am reproducing your article recently, and I have some doubts about the result, so I would like to consult you. Thank you very much if you can have time to answer!@Mehooz

I'd like to ask you,
1. Is the training <episode return> in the fourth figure in the article consistent with the <Running_Average_Rewards> in the code? Why can't my training results match?

2. In addition, is  the Hyperparameter name 《of samples of per iteration》  in the text  《epoch_frame》  in the training code configuration file?

3. Why do we need to interact with four actions each time we interact with the environment, but <Running_Average_Rewards> in the training code only calculates the average reward of the four actions? Figure 4 in the article is the average reward of the four actions or the sum result?
Thank you very much for your time and answer
